From b1996b49fb3839665f1770c8ec35a66f6caa6c6c Mon Sep 17 00:00:00 2001
From: Andrew Boie <andrew.p.boie@intel.com>
Date: Thu, 23 May 2013 12:14:34 -0700
Subject: include iago provision.mk

This needs to be included after the definitions for the
target-files-package.

Change-Id: I42a5e3a56405df52f8d2b6388e0cc2479e4d4de5
Signed-off-by: Andrew Boie <andrew.p.boie@intel.com>
---
 core/Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/core/Makefile b/core/Makefile
index f89e8e6..e088d7b 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1693,6 +1693,7 @@ $(INTERNAL_FINDBUGS_HTML_TARGET): $(INTERNAL_FINDBUGS_XML_TARGET)
 include $(sort $(wildcard $(BUILD_SYSTEM)/tasks/*.mk))
 -include $(sort $(wildcard vendor/*/build/tasks/*.mk))
 -include $(sort $(wildcard device/*/build/tasks/*.mk))
+-include bootable/iago/provision.mk
 
 # -----------------------------------------------------------------
 # Create SDK repository packages. Must be done after tasks/* since
-- 
1.7.4.1

